integration-tests/config_template.yaml

```diff
@@ -100,8 +100,8 @@ static_resources:
                       },
                       "services": [
                         {
-                          "id": {{or .ServiceID "\"test-service-id\""}},
-                          "token": {{or .ServiceToken "\"test-service-token\""}},
+                          "id": "{{or .ServiceID "\"test-service-id\""}}",
+                          "token": "{{or .ServiceToken "\"test-service-token\""}}",
```

Comment:
Note here that default case will return `""test-service-id""` and `""test-service-token""`

Reply:
okay will fix. I think we don't need a default since we use UUIDs